monin_obukhov/monin_obukhov_kernel.F90

```diff
@@ -18,7 +18,7 @@
 !***********************************************************************
 
 
-module monin_obukhov_inter
+module monin_obukhov_kernel
```

Comment:
Why change the name of this module?  monin_obukhov is used by land and atmosphere.  Will this have any effect on their code?

Reply:
I couldn't get the .mod file to copy over to the common directory if the name of the module was different from the name of the source file.  I changed the module name back to what it was.  Instead, I changed the name of the file to be the same as the name of the module.